json-specs/objective-spec.json

```diff
@@ -566,6 +627,38 @@
         ],
         "doc": "Lp Norm of elastic stress over the volume selection"
     },
+    {
```

Comment:
not now, but maybe we should split each objective in its own separate json

Reply:
Yea, maybe this is best...